src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/FrameResponseStream.cs

```diff
@@ -31,6 +34,7 @@ public override long Position
             set => throw new NotSupportedException();
         }
 
+        // REVIEW: Should Flush() throw?
```

Comment:
Which will call Write and throw: https://github.com/dotnet/corefx/blob/6dd451f51451a7d0ceea6104b51bd17005e9a0e6/src/System.Runtime.Extensions/src/System/IO/StreamWriter.cs#L253